### PR TITLE
"001_map_equidist_EQ": Mention alias "lakes" for "C" of "Figure.coast"

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
This PR aims to fix a few typos in the GMT scripts and Jupyter notebook in the folders "001_map_equidist_EQ" & "002_map_equidist_EQ_GCMT".

**gitnotebooks**: https://gitnotebooks.com/michaelgrund/GMT-plotting/pull/16

### CODE REVIEW

**Comment by yvonnefroehlich** [comment]
Regarding the Jupyter notebook [001_map_equidist_EQ/pygmt_jupyter_notebook/pygmt_plot_equidist_EQs.ipynb](https://github.com/michaelgrund/GMT-plotting/blob/main/001_map_equidist_EQ/pygmt_jupyter_notebook/pygmt_plot_equidist_EQs.ipynb):
Sorting the events descending by magnitude would avoid hiding a lot of small events by larger events:

- directly within the USGS data query `order = 'orderby=magnitude' `  (please note this impacts on the histogram)
- afterwards using pandas `data.sort_values(by=['mag'], ascending=False)`

 Maybe it makes sense to use a lighter / white outline for a better contrast.

| no sorting by magnitude | descending sorting by magnitude |
| --- | --- |
| PLOT_EQglob_BFO_black | PLOT_EQglob_BFO_black_magdescsorting |
|  PLOT_EQglob_BFO_white | PLOT_EQglob_BFO_white_magdescsorting |

File: 001_map_equidist_EQ/GMT_EQglob_BFO.gmt [hunk 1]
```diff
@@ -64,7 +64,7 @@ cmap2use=viridis.cpt
 
 
 # inferno_mod.cpt, magma_mod.cpt and plasma_mod.cpt are slightly modified versions of 
-# inferno.cpt, magma.cpt and plasma.cpt that can be also download from:
```
Comment (yvonnefroehlich) [inline 001_map_equidist_EQ/GMT_EQglob_BFO.gmt:67]: Based on this update in the README https://github.com/michaelgrund/GMT-plotting/blame/8dfbf9620a343d821d9f4cda76c1e7380d09094f/README.md#L3

> Updates for PyGMT based notebooks are carried out at irregular intervals. The GMT bash scripts are not maintained anymore.

Please indicate if you prefer to reduce this PR to the change in the related Jupyter notebook.

Comment (michaelgrund) [reply 001_map_equidist_EQ/GMT_EQglob_BFO.gmt:67]: Yes would be fine to just adjust the JN and leave the bash script as is.

Comment (yvonnefroehlich) [reply 001_map_equidist_EQ/GMT_EQglob_BFO.gmt]: I have reverted the changes in the two bash scripts in commit https://github.com/michaelgrund/GMT-plotting/pull/16/commits/d0187b4be4b9733683c0e273133797d0393ae477. After this, this PR is reduced to only mentioning the alias `lake` for `C` of `Figure.coast`.

**Review by michaelgrund — APPROVED** [review]

**Pull request merged**